components/resources/customize_toolbar_ui_strings.grdp

```diff
@@ -21,4 +21,7 @@
   <message name="IDS_CUSTOMIZE_TOOLBAR_TOGGLE_REWARD" desc="Title for a toggle button to show or hide rewards">
     Rewards
   </message>
+  <message name="IDS_CUSTOMIZE_TOOLBAR_TOGGLE_BRAVE_NEWS" desc="Title for a toggle button to show or hide brave news">
+    Add to Brave News
```

Comment:
I think this string is wrong - should it be "Show Brave News" or something?

Reply:
@mkarolin sure attached screenshot in the pr body.

This controls "add to brave news" button in the address bar, so I think it's fine as is.

<img width="495" height="48" alt="image" src="https://github.com/user-attachments/assets/864940c8-e053-4d09-a7ea-b6b87b7cc4f9" />
^ rss button in the screenshot